freeipa/src/main/resources/freeipa-salt/salt/logrotate/conf/pkidebug

```diff
@@ -0,0 +1,12 @@
+/var/log/pki/pki-tomcat/ca/debug {
+    missingok
+    notifempty
+    create 0640 pkiuser pkiuser
+    daily
+    size 128M
+    rotate 6
+    su pkiuser pkiuser
+    nocompress
+    postrotate
+        /bin/systemctl restart pki-tomcatd@pki-tomcat > /dev/null 2>/dev/null || true
```

Comment:
I am thinking this just will impact the CA & CRL creation for a short time. Do you know if it impacts anything else?

Reply:
please try with `copytruncate` instead of restarting the service: https://linux.die.net/man/8/logrotate

Reply:
Thanks! I tested copytruncate. Now it is working good without restart the service.